ansible/roles/cloudflared/tasks/install.yml

```diff
@@ -45,4 +45,5 @@
     src: "{{ tempfolder.path }}/cloudflared"
     remote_src: yes
     dest: /usr/local/bin/cloudflared
+    mode: '0777'
```

Comment:
I'd change this to `0755`, no reason for users other than the owner to write this file, and especially no reason in making it world writable (security vulnerability).

Reply:
Agreed, just lowered to 0755 👍